CHANGELOG.md

```diff
@@ -4,120 +4,18 @@
 
 [Full Changelog](https://github.com/shopsys/shopsys/compare/v10.0.4...v9.1.3)
 
-**Merged pull requests:**
-
-- \[framework\] update heureka/overeno-zakazniky to latest version 4.0.1 [\#2534](https://github.com/shopsys/shopsys/pull/2534) ([RostislavKreisinger](https://github.com/RostislavKreisinger))
```

Comment:
I asked you to fill missing changes in 9.1.3 and you even removed the only one mentioned :D